source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -277,18 +277,41 @@ def wrapper(*args, **kwargs):
                         if not app_value:
                             continue
 
+                        # Get the territory value for the line used
+                        territory_value = None
+                        line_id = next(
+                            (link['id'] for link in s.get('links', []) if link['type'] == "line"), None
+                        )
+                        if line_id:
+                            line_details = instance.ptref.get_objs(
+                                type_pb2.LINE, 'line.uri=\"{}\"'.format(line_id), type_pb2.OdtLevel.all
```

Comment:
Do you really need that odt filter?

As it's available in `filter`, I really think we shouldn't change the interface and just complete filter before call with something like: ` and line.odt_level(<level>)`
see https://github.com/hove-io/navitia/blob/dev/documentation/rfc/ptref_grammar.md

Reply:
After discussion : :disappointed: not possible because of https://github.com/hove-io/navitia/blob/693a1de8c88d57a771f6808a87e3e1ad58011a7a/source/kraken/worker.cpp#L192 (defaulting to `scheduled` lines).
Little improvement of interface to come anyway :pray:

Reply:
:+1: